### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#182] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252438-b5d69d69-4b05-4bd0-acef-03c60dceeb66.png)

**Comment by Exyss** [comment]
1. $26^2 \cdot 10^3 \cdot 26 \cdot 1$
2. $25^2 \cdot 10^3 \cdot 25 \cdot 1$
3. $Tot - NessunaC = (26^2 \cdot 10^3 \cdot 26^2) - (25^2 \cdot 10^3 \cdot 25^2)$
4. $4 \cdot (25^2 \cdot 10^3 \cdot 25 \cdot 1)$
5. $FinisconoZ - NessunaCeFinisconoZ = (26^2 \cdot 10^3 \cdot 26 \cdot 1) - (25^2 \cdot 10^3 \cdot 25 \cdot 1)$
6. $D_{4,2} * (24^2 * 10^3) = \dfrac{4!}{2!} (24^2 \cdot 10^3) = 12(24^2 \cdot 10^3)$

**Comment by matypist** [comment]
Propongo un altro metodo per la risoluzione del 5° punto che **porta allo stesso risultato**, dunque possiamo dire la soluzione al 5° punto **confermata per Doppio Conteggio**

-----------------------

Divido il complemento "non hanno nessuna C e/o non finiscono con Z" in 3 tipi mutualmente disgiunti ed esaustivi:
**T1** = {Hanno almeno una C e non finiscono con Z}
**T2** = {Non hanno nessuna C e finiscono con Z}
**T3** = {Non hanno nessuna C e non finiscono con Z}

O, se vogliamo, in 2 tipi (unendo il secondo e il terzo):
**T1** = {Hanno almeno una C e non finiscono con Z} => **#T1** = {Totale delle targhe che non finiscono con Z} - {Totale delle targhe che non hanno nessuna C e non finiscono con Z) =  (26 * 26 * 10 ^ 3 * 26 * 25) - (25 * 25 * 10^3 * 25 * 24)
**T2** = {Non hanno nessuna C} => **#T2** = (25 ^ 2 * 10^3 * 25 * 25)

Per poi sottrarre la loro somma al totale:
**#Tot** - (**#{T1}** + **#{T2}**) = **#Tot** - **#{T1}** - **#{T2}** = (26^2 * 10^3 * 26^2) - [(26 * 26 * 10 ^ 3 * 26 * 25) - (25 * 25 * 10^3 * 25 * 24)] - (25 ^ 2 * 10^3 * 25 * 25)

**Comment by VictorLopata** [comment]
La 6 credo sia tipo: 12 * (24^2 * 10^3) e non 12 * (24^2 * 10^3 *24^2 )

**Comment by matypist** [comment]
Mi unisco ad altre due persone (vedi reactions di pollice in sù) nel confermare per il punto 6 la soluzione di vikkloppp, nella soluzione per il punto 6 di @Exyss vi era probabilmente un mero refuso nel suo moltiplicare una volta di troppo $24^2$
